src/elfcore.c

```diff
@@ -923,6 +934,17 @@ static int CreateElfCore(void *handle,
                       }
                     }
 
+                    #if defined(__aarch64__)
+                    /* check if the flag is "pf" */
```

Comment:
This comment is not helpful since it's obvious that this is what the code does.  Better would be to say WHY we check for this; apparently we don't dump segments with the "pf" flag for some reason?